src/Admin/AdminInterface.php

```diff
@@ -215,181 +176,130 @@ public function getIdParameter();
 
     /**
      * Returns true if the route $name is available.
-     *
-     * @param string $name
-     *
-     * @return bool
      */
-    public function hasRoute($name);
+    public function hasRoute(string $name): bool;
 
-    public function setSecurityHandler(SecurityHandlerInterface $securityHandler);
+    public function setSecurityHandler(SecurityHandlerInterface $securityHandler): void;
 
-    /**
-     * @return SecurityHandlerInterface|null
-     */
-    public function getSecurityHandler();
+    public function getSecurityHandler(): ?SecurityHandlerInterface;
 
     /**
-     * @param string      $name
-     * @param object|null $object
-     *
-     * @return bool
+     * @param string $name
      */
-    public function isGranted($name, $object = null);
+    public function isGranted($name, ?object $object = null): bool;
 
     /**
      * @param mixed $entity
      *
      * @return string a string representation of the identifiers for this instance
      */
-    public function getNormalizedIdentifier($entity);
+    public function getNormalizedIdentifier($entity): ?string;
 
     /**
      * Shorthand method for templating.
      *
      * @param object $entity
-     *
-     * @return mixed
      */
-    public function id($entity);
+    public function id($entity): ?string;
```

Comment:
```suggestion
    public function id(object $entity): ?string;
```

Reply:
This argument is passed to `getNormalizedIdentifier()` method at model manager, which seems to allow null.